<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addContact</class>
 <widget class="QWidget" name="addContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>436</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>436</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>add contact window</string>
  </property>
  <widget class="QLabel" name="note_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>заметка</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="surname_label">
        <property name="text">
         <string>фамилия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="phone_label">
        <property name="text">
         <string>телефон</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="email_label">
        <property name="text">
         <string>почта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="adress_label">
        <property name="text">
         <string>адрес</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="surname_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="phone_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="email_edit"/>
      </item>
      <item>
       <widget class="QTextEdit" name="adress_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>добавить контакт</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {background-color: rgb(205, 205, 205)};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set_image">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>выбрать фото</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="note_edit">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>189</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
